[Handheld Multi-Modal Application] Documentation Update (#3274)
open-edge-platform/edge-ai-suites: federal-aerospace/docs/handheld-multi-modal/user-guide/deploy-applications.md

c7e676f86e1b   intel/model-download:2026.1.0-20260505-weekly           "/opt/entrypoint.sh …"   34 seconds ago   Up 33 seconds (healthy)            0.0.0.0:8000->8000/tcp, [::]:8000->8000/tcp
```

## Accessing Application User Interface

This composite application exposes multiple endpoints that are accessible via the NGINX Transport Layer Security (TLS) reverse proxy and are bound to `localhost` address only (are not exposed under any

> external IP address). As the intended use in on handheld devices, the applications do not need to provide authentication or authorization.

Reviewer: ```suggestion
This composite application exposes multiple endpoints through the NGINX TLS reverse proxy. They are bound to localhost only and are not exposed on any external IP address. 
Since the intended use is on handheld devices, the applications do not provide authentication or authorization.
```

> external IP address). As the intended use in on handheld devices, the applications do not need to provide authentication or authorization.

Reviewer 2: ```suggestion
This composite application exposes multiple endpoints that are accessible via the NGINX
Transport Layer Security (TLS) reverse proxy and are bound to the `localhost` address
only (are not exposed under any external IP address). As the application is intended for
handheld devices, it does not need to provide authentication or authorization.
```

> **Notice**:
> A browser warning about use of self-signed certificate is an expected behavior as the NGINX reverse proxy uses such certificate to assure TLS transport on the `localhost` bound addresses.

> > Supporting HTTPS connections is required on modern browsers to enable microphone input used by Open WebUI and Speech To Text services.

Reviewer: ```suggestion
> A browser warning about the self-signed certificate is expected — the NGINX reverse proxy uses one to provide TLS on the localhost-bound addresses.
> HTTPS is required by modern browsers to enable the microphone input used by Open WebUI and Speech To Text services.
```

> > A browser warning about use of self-signed certificate is an expected behavior as the NGINX reverse proxy uses such certificate to assure TLS transport on the `localhost` bound addresses.
> > Supporting HTTPS connections is required on modern browsers to enable microphone input used by Open WebUI and Speech To Text services.

Reviewer 2: ```suggestion
> A browser warning about the use of a self-signed certificate is an expected behavior.
> Modern browsers require HTTPS to enable microphone input used by Open WebUI and Speech To Text services, therefore, the NGINX reverse proxy uses the certificate to ensure TLS transport on the `localhost` bound addresses.
```
